nist/int.go

```diff
@@ -346,6 +341,21 @@ func (i *Int) BigEndian(min, max int) []byte {
 	return buf
 }
 
+// Set value to a number represented by a byte string.
+// Endianness depends on the endianess set in i.
+func (i *Int) SetBytes(a []byte) *Int {
+	i.V.SetBytes(a).Mod(&i.V, i.M)
+	return i
+}
+
+// Set value to a number represented in a little-endian byte string.
+func (i *Int) SetLittleEndian(a []byte) *Int {
```

Comment:
I also think that this should not be needed anymore but when I asked you you said "let's keep it" .. :) I'm all for removing that !